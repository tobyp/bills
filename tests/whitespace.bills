#!./bills
A B. 1.0
   C        D. 2.0       
	E 	 F.		3.0
G	H. 4.0
I J. 2.0		
