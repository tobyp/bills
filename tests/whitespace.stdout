todo: B pays 1.0 to A
todo: D pays 2.0 to I
todo: F pays 3.0 to E
todo: H pays 4.0 to G
todo: J pays 2.0 to C
